dev-client/src/components/util/transformNativeBase.ts

```diff
@@ -15,62 +15,107 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
```

Comment:
question: what does this file do? is it temporary?

Reply:
it is a dev-time script to do structured code analysis/refactoring, sort of temporary. it's how i've been tracking how many native base components are left and automatically rewriting import statements. there's no real harm to having it in the `src` tree so it didn't seem worth it to figure out how to have it somewhere else. i will either delete it or move it out of `src` when the native base migration is done